src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (x = 0) => {
+    count++;
+
+    return (y = 0) => {
+      return (count > 3 && count % 2 === 0)
```

Comment:
You could make it look prettier 
```suggestion
  return (x) => {
    return (y)  => {
      count++;
    
      return (count > 3 && count % 2 === 0)
```

Reply:
good point from @Iksich 👍